bindings/stronghold-nodejs/rustfmt.toml

```diff
@@ -0,0 +1,9 @@
+comment_width = 120
```

Comment:
I think in the Wasm binding this is linked to the root config, we should probably do the same here.